etc/schema.json

```diff
@@ -3669,126 +3669,165 @@
                                 },
                                 "ftp-data": {
+                                    "description": "Errors encountered parsing FTP data",
                                     "$ref": "#/$defs/stats_applayer_error"
                                 },
                                 "http": {
+                                    "description": "Errors encountered parsing HTTP",
                                     "$ref": "#/$defs/stats_applayer_error"
                                 },
                                 "http2": {
+                                    "description": "Errors encountered parsing HTTP 2",
                                     "$ref": "#/$defs/stats_applayer_error"
                                 },
                                 "ike": {
+                                    "description": "Errors encountered parsing IKE protocol",
                                     "$ref": "#/$defs/stats_applayer_error"
                                 },
                                 "imap": {
+                                    "description": "Errors encountered parsing IMAP",
                                     "$ref": "#/$defs/stats_applayer_error"
                                 },
                                 "krb5_tcp": {
+                                    "description": "Errors encountered parsing Kerberos5/TCP protocol",
                                     "$ref": "#/$defs/stats_applayer_error"
                                 },
                                 "krb5_udp": {
+                                    "description": "Errors encountered parsing Kerberos5/UDP protocol",
                                     "$ref": "#/$defs/stats_applayer_error"
                                 },
                                 "modbus": {
+                                    "description": "Errors encountered parsing Modbus protocol",
                                     "$ref": "#/$defs/stats_applayer_error"
                                 },
                                 "mqtt": {
+                                    "description": "Errors encountered parsing MQTT protocol",
                                     "$ref": "#/$defs/stats_applayer_error"
                                 },
                                 "nfs_tcp": {
+                                    "description": "Errors encountered parsing NFS/TCP protocol",
                                     "$ref": "#/$defs/stats_applayer_error"
                                 },
                                 "nfs_udp": {
+                                    "description": "Errors encountered parsing NFS/UDP protocol",
                                     "$ref": "#/$defs/stats_applayer_error"
                                 },
                                 "ntp": {
+                                    "description": "Errors encountered parsing NTP",
```

Comment:
most entries have "<proto> _protocol_", would be good to be consistent

Reply:
I skipped it in places where the protocol's abbreviation ended in `P` that stood for protocol. Like here. Else it becomes _Network Time Protocol_ protocol.
If you think adding a protocol is more convenient to read, lmk.

Reply:
ah, makes sense now, thanks